stdlib/LinearAlgebra/test/generic.jl

```diff
@@ -248,6 +248,18 @@ end
     end
 end
 
+@testset "normalize for multidimensional arrays" begin
+    arr = [ [1.0 0.0]; [0.0 1.0] ]
+    @test normalize(arr) == normalize!(copy(arr))
+
+    arr = [
+            [1.0 0.0 0.0];
+            [0.0 1.0 0.0]
+         ]
+    @test normalize(arr) == normalize!(copy(arr))
+
```

Comment:
It would be good to test that this is consistent with the vector case. Some tests like
```julia
arr = randn(2,3)
@test size(normalize(arr)) == size(arr)
@test vec(normalize(arr)) == normalize(vec(arr))
```

Reply:
added a test case based on a testcase I found in test/offset.jl

```
julia> A = OffsetArray([-1,0], (-2,))
2-element OffsetArray{Int64,1,Array{Int64,1}} with indices -1:0:
 -1
  0
julia> Arsc = reshape(A, :, 1)
2×1 reshape(::OffsetArray{Int64,1,Array{Int64,1}}, 2, 1) with eltype Int64:
 -1
  0
julia> Arsc[1,1] = 5
5
julia> first(A)
5
julia> A[1]
ERROR: BoundsError: attempt to access 2-element OffsetArray{Int64,1,Array{Int64,1}} with indices -1:0 at index [1]
```
Works with normalize now:
```
julia> normalize(A)
2-element OffsetArray{Float64,1,Array{Float64,1}} with indices -1:0:
 1.0
 0.0
```